DEVELOPING.md

```diff
@@ -46,18 +46,17 @@ applications:
     charm: /tmp/charm-builds/canal
     resources:
       calico: /path/to/layer-canal/calico-amd64.tar.gz
-      calico-upgrade: /path/to/layer-canal/calico-upgrade-amd64.tar.gz
       flannel: /path/to/layer-canal/flannel-amd64.tar.gz
 ```
 
 Then deploy Charmed Kubernetes with your locally built canal charm:
 
 ```
-juju deploy cs:~containers/canonical-kubernetes-canal --overlay local-canal.yaml
+juju deploy canonical-kubernetes-canal --overlay local-canal.yaml
```

Comment:
this bundle `canonical-kubernetes-canal` doesn't exist. 

I think you'd have to do `charmed-kubernetes` + some canal overlay

Reply:
You are correct that we would normally want c-k to be the "real" bundle, modified as needed with overlays.

However, there is a c-k-c bundle in ch. It made it over during the sync because it was a promulgated bundle in cs:

https://charmhub.io/canonical-kubernetes-canal

Reply:
🤮